rsmtool/configuration_parser.py

```diff
@@ -764,15 +766,15 @@ def _check_config_is_loaded(self):
 
 
     @classmethod
-    def get_configparser(cls, filepath, *args, **kwargs):
+    def get_configparser(cls, config_file_or_obj_or_dict, *args, **kwargs):
```

Comment:
Hmm, might it make more sense to just move this logic to the `ConfigurationParser()` initializer instead of having this classmethod since all we are doing is getting the args and then calling the initializer anyway?